### PR TITLE
让初次使用者能看得懂简体中文

### PR DESCRIPTION
**Pull request by xiaoyaofengjianshi**
优化了部分语言显示，新用户不会抓瞎了

### CODE REVIEW

**Comment by xueying2013** [comment]
support

File: 快速构建器/i18n/zh_CN.go [hunk 1]
```diff
@@ -5,10 +5,10 @@ var I18nDict_zh_CN map[uint16]string = map[uint16]string{
 	ACME_FailedToSeek:                   "无效ACME文件，因为seek操作失败了。",
 	ACME_StructureErrorNotice:           "文件结构错误",
 	ACME_UnknownCommand:                 "未知ACME命令（文件错误）",
-	Auth_BackendError:                   "后端错误",
+	Auth_BackendError:                   "后端错误，别担心，这不是你的问题",
 	Auth_FailedToRequestEntry:           "未能请求租赁服入口，请检查租赁服等级设置是否关闭及租赁服密码是否正确。",
-	Auth_HelperNotCreated:               "辅助用户尚未创建，请前往用户中心进行创建。",
-	Auth_InvalidFBVersion:               "FastBuilder 版本无效，请更新。",
+	Auth_HelperNotCreated： 		    "辅助用户尚未创建，请前往用户中心进行创建。",
```
Comment (LNSSPsd) [inline 快速构建器/i18n/zh_CN.go:10]:
```suggestion
	Auth_HelperNotCreated: 		    "辅助用户尚未创建，请前往用户中心进行创建。",
```
There is an invalid syntax.

**Review by LNSSPsd — CHANGES_REQUESTED** [review]
Whether this modification is necessary is to be questioned, but however there is an invalid syntax.

**Comment by LNSSPsd** [comment]
And you are trying to rename the zh_CN i18n file to an unexpected place, which further break it harder.

File: 快速构建器/i18n/zh_CN.go [hunk 2]
```diff
@@ -5,11 +5,11 @@ var I18nDict_zh_CN map[uint16]string = map[uint16]string{
...
 	ACME_UnknownCommand:                 "未知ACME命令（文件错误）",
-	Auth_BackendError:                   "后端错误",
+	Auth_BackendError:                   "后端错误，别担心，这不是你的问题",
 	Auth_FailedToRequestEntry:           "未能请求租赁服入口，请检查租赁服等级设置是否关闭及租赁服密码是否正确。",
-	Auth_HelperNotCreated:               "辅助用户尚未创建，请前往用户中心进行创建。",
-	Auth_InvalidFBVersion:               "FastBuilder 版本无效，请更新。",
-	Auth_InvalidHelperUsername:          "辅助用户的用户名无效，请前往用户中心进行设置。",
+	Auth_HelperNotCreated： 		    "辅助用户尚未创建，请前往用户中心进行创建。",
+	Auth_InvalidFBVersion: 	             "Fastbuilder版本不是最新版，请更新或用'--no-update-check'启动。",
+	Auth_HelperNotCreated： 		    "辅助用户尚未创建，请前往用户中心进行创建。"
```
Comment (LNSSPsd) [inline 快速构建器/i18n/zh_CN.go:12]:
```suggestion
```
Unexpected stuff is being left everywhere

**Review by LNSSPsd — CHANGES_REQUESTED** [review]
And invalid filename remains

**Review by LNSSPsd — CHANGES_REQUESTED** [review]

**Comment by RainyHallways** [comment]
Line 41: Copyright_Notice_Bouldev:            "版权所有 (c) FastBuilder DevGroup, Bouldev 2022",
should change to: Copyright_Notice_Bouldev:            "版权所有 (c) FastBuilder DevGroup, Bouldev 2023", or not?

**Pull request closed without merging**